docs/api-docs/catalog/v2-vs-v3.md

```diff
@@ -0,0 +1,128 @@
+
+Variant example: large blue t-shirt.
+
+#### Modifiers
+[Modifiers](https://developer.bigcommerce.com/api-docs/store-management/products-overview#modifier-options) represent options used for additional product customization such as gift wrapping, engraving, text to be printed on a t-shirt, or a warranty. Unlike variant options, modifiers do not generate variants and can be displayed as *required* or *optional* on the storefront. You can use modifiers along with [rules](https://developer.bigcommerce.com/api-docs/store-management/products-overview#complex-rules) to change a product's price or weight.
+Modifiers include multiple-choice types such as swatch, rectangle, radio button, and dropdown and non-multiple-choice types such as text, multi-line text, date picker, file upload, and pick list.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+>
+> Product [variant options](https://developer.bigcommerce.com/api-reference/store-management/catalog/product-options/getoptionbyid) and [modifiers](https://developer.bigcommerce.com/api-reference/store-management/catalog/product-modifiers/getmodifierbyid) created using the V3 Catalog API belong to a single product instance and cannot be attached to other products in the catalog.
+
+</div>
+</div>
+</div>
+
+#### Creating variants using V2 and V3 Catalog API
+
+The following workflows demonstrate the difference between creating a product with variants using V2 and V3 Catalog APIs.
+
+**V2 API workflow to create a product with variants:**
+
+1. Create a product using the V2 Products API.
+2. Create the options.
+3. Create an option set using the V2 Option Sets API.
+4. Link the product to the option set.
+5. Create adjustments, such as price adjustment, using rules.
+
+**V3 Catalog API workflow to create a product with variants:**
+
+1. [Create a product](https://developer.bigcommerce.com/api-reference/store-management/catalog/products/createproduct) with variants by sending a `POST` request to `/v3/catalog/products`. You can specify the price directly on the variant level using the `price` property of the variant object.
+
+#### Product variants
+
+In V3, every purchasable entity in the catalog is a variant including the product itself. This enables enhanced inventory management flows such as the ability to use the [variants](https://developer.bigcommerce.com/api-reference/store-management/catalog/variants/getvariants) endpoint to manage inventory levels. While it is possible to create a SKU with a subset of product options using the V2 API; in V3, variants must be created for every combination of option values. We recommend creating products using the V3 API as BigCommerce intends to move operations to the V3 API. 
```

Comment:
In V3, every purchasable entity in the catalog is a variant including the product itself. This structure enables enhanced inventory management flows, such as using the [variants](https://developer.bigcommerce.com/api-reference/store-management/catalog/variants/getvariants) endpoint to manage inventory levels. While it is possible to create an SKU with a subset of product options using the V2 API; in V3, you must create variants for every combination of option values. We recommend creating products using the V3 API as BigCommerce intends to move operations to the V3 API.